scripts/_0_0_10_fix_participant_id.py

```diff
@@ -0,0 +1,24 @@
+def fix_participant_id(mongoDB):
```

Comment:
If the DB version is 0.1.0, presumably this filename should be `_0_1_0_fix_participant_id.py`?